build.gradle

```diff
@@ -136,27 +161,23 @@ task lobbyServer(type: Zip, group: 'release', dependsOn: ':shadowJar') {
 	from(configurations.testCompile.files { dep -> dep.name == 'derby' }) {
 		into('lib')
 	}
-	from("build/libs/triplea-${project.version}-all.jar") {
+	from(renameShadowJar.output) {
 		into('bin')
-		rename "triplea-${project.version}-all.jar", 'triplea.jar'
 	}
 }
 
 task generateZipReleases(group: 'release', dependsOn: [allPlatform, bots, lobbyServer]) {}
 
-task shadowJarCopy(type: Copy) {
-	from "build/libs/triplea-${project.version}-all.jar"
-	into 'build/libs'
-	rename { String fileName ->
-		fileName.replace("triplea-${project.version}-all.jar", "triplea.jar")
-	}
-}
-
-
-import com.install4j.gradle.Install4jTask
 apply plugin: 'install4j'
 
-task generateInstallers(type: Install4jTask, dependsOn: [ ':shadowJar', shadowJarCopy ],  group: 'release') {
+task copyJRE(type: Copy, dependsOn: [updateAssets]) {
+	from "${assetsDirectory}/install4j/windows-x86-1.8.0_66.tar.gz"
```

Comment:
If you know the syntax, it does seem like these JREs should be listed in an array or a list, and then a for loop down here to loop over each one. Mainly trying to get the hardcoded values to be extracted. Not a big deal on this one. address this comment last if at all.